### PR TITLE
added PubNub as-a-Service

### PR DESCRIPTION
**Pull request by stephenlb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -843,7 +844,6 @@ Table of Contents
    * [kanbanflow.com](https://kanbanflow.com/) — Board-based project management. Free, premium version with more options
    * [kanbantool.com](https://kanbantool.com/) — Kanban board-based project management. Free, paid plans with more options
    * [kanrails.com](https://kanrails.com/) — Kanban board-based project management. Free for 3 collaborators, 2 projects and 5 tracks. Paid plans available for unlimited collaborators, projects and tracks.
-   * [Kitemaker.co](https://kitemaker.co) - Collaborate through all phases of the product development process and keep track of work across Slack, Discord, Figma, and Github. Unlimited users, unlimited spaces. Free plan up to 250 work items.
```
Comment (ripienaar) [inline README.md:846]: this one should stay (just merged, perhaps some weird conflict here)

Comment (stephenlb) [reply README.md:846]: I was wondering about this, closed/reopened PR

Comment (stephenlb) [reply README.md:846]: Thought I accidentally deleted that line

Comment (stephenlb) [reply README.md:846]: lmk we can recreate new PR

Comment (ripienaar) [reply README.md:846]: You can also just send a 2nd commit to the same PR to put it back, thats fine too - which is easier for you, new PR or update this one

Comment (stephenlb) [reply README.md:846]: on it

Comment (stephenlb) [reply README.md:846]: done - https://github.com/ripienaar/free-for-dev/pull/1812/commits/f41a5a310fcde6fe4ec002abe1367b05cb5a4abc

Comment (stephenlb) [reply README.md:846]: added - https://github.com/ripienaar/free-for-dev/commit/f41a5a310fcde6fe4ec002abe1367b05cb5a4abc

image
image

**Comment by ripienaar** [comment]
thank you :)

**Pull request merged**